### DISCUSSION TITLE
Add ability to Attach PDF reports to cables and Circuits.

### DISCUSSION THREAD

**Discussion by lucasw-eng** [netbox-community/netbox#9465] [Ideas]
I was thinking it would be nice to have the ability to attach PDF reports from Link runner and OTDR tests to the defined cable. Also when requesting a L2 circuit from a provider they typically provide an RFC 2544 report which would be nice to be able to attach to the respective circuit.

**Comment by matejv** [comment]
There is a plugin [netbox-documents](https://github.com/jasonyates/netbox-documents) that might do what you need.

**Reply by lucasw-eng** [reply]
Thanks for this, I'll look into the plugin at first glance it partially meets my needs. However, doesn't look like it supports attaching documents to cables yet.
